### ISSUE TITLE
run train.sh failed

### ISSUE THREAD

**Issue by OneMondy** [XingangPan/SCNN#36]
Hi xingang,
After running train.sh, the following error is encountered: (the data set has been checked, part of the driver_182_30 data set is used as the training set, and the other part is used as the validation set) run train.hope you can answer for me.thank you!

load file: /home/hfut/Documents/ScnnNew/data/CULane/list/train_gt.txt
/home/hfut/torch/install/bin/luajit: /home/hfut/Documents/qiao/ScnnNew/datasets/lane-gen.lua:18: attempt to index local 'f' (a nil value)
stack traceback:
        /home/hfut/Documents/qiao/ScnnNew/datasets/lane-gen.lua:18: in function 'getPaths'
        /home/hfut/Documents/qiao/ScnnNew/datasets/lane-gen.lua:103: in function 'exec'
        ./datasets/init.lua:28: in function 'create'
        ./dataloader.lua:31: in function 'create'
        main.lua:33: in main chunk
        [C]: in function 'dofile'
        ...hfut/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
        [C]: at 0x00405d50
@XingangPan

**Comment by aquariusnk** [comment]
> Hi xingang,
> After running train.sh, the following error is encountered: (the data set has been checked, part of the driver_182_30 data set is used as the training set, and the other part is used as the validation set) run train.hope you can answer for me.thank you!
> 
> load file: /home/hfut/Documents/ScnnNew/data/CULane/list/train_gt.txt
> /home/hfut/torch/install/bin/luajit: /home/hfut/Documents/qiao/ScnnNew/datasets/lane-gen.lua:18: attempt to index local 'f' (a nil value)
> stack traceback:
> /home/hfut/Documents/qiao/ScnnNew/datasets/lane-gen.lua:18: in function 'getPaths'
> /home/hfut/Documents/qiao/ScnnNew/datasets/lane-gen.lua:103: in function 'exec'
> ./datasets/init.lua:28: in function 'create'
> ./dataloader.lua:31: in function 'create'
> main.lua:33: in main chunk
> [C]: in function 'dofile'
> ...hfut/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
> [C]: at 0x00405d50
> @XingangPan

I also have the same issue. Have you found a solution yet? @OneMondy
